tests/sentry/helpers/test_deprecation.py

```diff
@@ -89,3 +99,60 @@ def test_self_hosted(self):
 
     def test_no_decorator(self):
         self.assert_not_deprecated("HEAD")
+
+    def test_default_key(self):
+        settings.SENTRY_OPTIONS.update({"api.deprecation.brownout-cron": custom_cron})
+        settings.SENTRY_OPTIONS.update({"api.deprecation.brownout-duration": custom_duration})
+
+        custom_timeiter = croniter(custom_cron, test_date)
```

Comment:
nit: I think it's a bit more ledgible as `custom_time_iter`, even if the package doesn't use `cron_iter`